src/qml/components/Footer.qml

```diff
@@ -5,13 +5,17 @@ import QtQuick.Controls 2.12
 import "../theme"
 
 Item {
+    property string text: ""
+    readonly property real tightLetterSpacing: 0.23
+
     height: 40
     width: parent.width
+    visible: text !== "" ? true : false
 
     Text {
         anchors.centerIn: parent
-        text: qsTr("Powered by Virgil Security, Inc.")
-        font.letterSpacing: 0.23
+        text: text
```

Comment:
Use alias for this property.
`text: text` doesn't look good

Reply:
Ok